VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1829,4 +1830,13 @@ private void showWifiSettingsMessage() {
             logger.error(ex);
         }
     }
+
+    /**
+     * This method returns true if message is displayed on player to change wifi settings.
+     * @return
+     */
+    public boolean isShownWifiSettingsMessage(){
+        return curMessageTypes.contains(VideoNotPlayMessageType.IS_SHOWN_WIFI_SETTINGS_MESSAGE);
+    }
+
```

Comment:
The flag is cleared in <code>hideNetworkError<code> and initially when PlayerFragment is initialized. I haved tested this and it addresses the issue. 
Merging this.